lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -583,6 +584,20 @@ module.exports = class MethodMappings {
           return value;
         },
 
+        async isElementPresent(webElement) {
+          try {
+            // New element apis contains only webElement argument
+            const element = await webElement;
+
+            return element instanceof WebElement || element instanceof ShadowRoot;
+          } catch (error) {
+            if (error.name === 'NoSuchElementError') {
+              return false; // Element does not exist
+            }
+            throw error; // Re-throw other errors for further handling
+          }
```

Comment:
In what cases will this `catch` block be hit?

Because the `webElement` in the new Element API is designed to never actually throw an error in normal cases (including when element is not found), I don't think `try-catch` is required here. For other errors, they would just be thrown directly.

Is there any case that I am missing here in which this `catch` block will be useful?

Reply:
I checked a few cases and yes it was not catching any errors. removed.